Library/Formula/squirrel-sql.rb

```diff
@@ -0,0 +1,43 @@
+class SquirrelSql < Formula
+  desc "GUI for JDBC-compliant databases"
+  homepage "http://www.squirrelsql.org/"
+  head "git://git.code.sf.net/p/squirrel-sql/git"
```

Comment:
There are stable releases; it was just easier for me to start development on the HEAD rather than a stable release. (Squirrel's website is not particularly well organized - it can be a little tricky to find what you're looking for)